_pages/interview.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: page
```

Comment:
I'm not sure, but how about make the interview page merged with the position page (or another option is to put it into Info section via `posts`? For the following reasons:
1. this infomration is highly correlated to that one
2. we already have many navigation pages

Reply:
I merged it with the positions page.